simphony/testing/abc_check_engine.py

```diff
@@ -25,6 +30,11 @@ def setUp(self):
         self.items = self.create_dataset_items()
 
     @abc.abstractmethod
+    def container_factory(self, name, mode='w'):
```

Comment:
Ok, it makes sense now. I have moved the tests back to ```test_h5_cuds.py``` and implemented @nathanfranklin  suggestions.